pkg/qemu/qemu.go

```diff
@@ -920,11 +920,13 @@ func Cmdline(ctx context.Context, cfg Config) (exe string, args []string, err er
 	args = append(args, "-chardev", fmt.Sprintf("socket,id=%s,path=%s,server=on,wait=off", qmpChardev, qmpSock))
 	args = append(args, "-qmp", "chardev:"+qmpChardev)
 
+	// virtserialport doesn't seem to work reliably: https://github.com/lima-vm/lima/issues/2064
+	// should go together with hostagent::New changes
```

Comment:
The comment is not clear. If it is needed to explain why the code is commented it can be removed with the commented code.